finish_release.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+. ./release.sh
+
+merge_release_candidate (){
+    echo "Merge release-candidate into release"
+    git checkout -t origin/release
+    git merge origin/release-candidate
+    git push
+
```

Comment:
Can you remove these blank lines before the `}` of each function, and add a blank line between each function?